src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -200,7 +200,8 @@ protected void scrollToElement(WebElement element, int offset) {
     JavascriptExecutor js = (JavascriptExecutor) driver;
     try {
       js.executeScript("var x = $(arguments[0]);"
-          + "window.scroll(0,parseInt(x.offset().top - arguments[1]));", element, offset);
+                       + "window.scroll(0,parseInt(x.offset().top - arguments[1]));", element,
+                       offset);
```

Comment:
this will looks better:
```
js.executeScript(
   "var x = $(arguments[0]);" + 
   "window.scroll(0,parseInt(x.offset().top - arguments[1]));",
   element,
   offset
);
```